### PR TITLE
Resilience metrics fix issue #8220

### LINKED ISSUE #8220: Follow-up to Resilience Metrics PR
Issue overview
--------------
A couple cleanups following the merge of #8100 for IO freeze. More detail [here](https://github.com/NREL/EnergyPlus/pull/8100#issuecomment-676754636)
 - Variables declared as `double` in multiple locations need to be `Real64`
 - New documentation TeX file is not included in the main table of contents, so it does not appear inside the final PDF.

### PR DESCRIPTION
**Pull request by xuanluo113**
Pull request overview
---------------------
Fixes #8220 

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [x] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: doc/engineering-reference/engineering-reference.tex [hunk 1]
```diff
@@ -304,4 +304,6 @@
 
 \input{src/special-modules-reporting/heat-emissions}
 
+\input{src/special-modules-reporting/resilience-metrics}
```
Comment (Myoldmopar) [inline doc/engineering-reference/engineering-reference.tex:307]: Yep, that should do it.  I'll check out the built version to verify.

File: src/EnergyPlus/OutputReportTabular.cc [hunk 2]
```diff
@@ -11252,18 +11252,18 @@ namespace OutputReportTabular {
 
     void WriteResilienceBinsTable(int const columnNum,
                                   std::vector<int> const &columnHead,
-                                  Array1D<std::vector<double>> const &ZoneBins)
+                                  Array1D<std::vector<Real64>> const &ZoneBins)
     {
-        std::vector<double> columnMax(columnNum, 0);
-        std::vector<double> columnMin(columnNum, 0);
-        std::vector<double> columnSum(columnNum, 0);
+        std::vector<Real64> columnMax(columnNum, 0);
```
Comment (Myoldmopar) [inline src/EnergyPlus/OutputReportTabular.cc:11257]: :+1:

**Review by Myoldmopar — APPROVED** [review]
Code changes appear to cover the issues.  I'll verify the built PDF for the include statement change, and CI would verify if there was anything wrong with the double/Real64 changes.

**Comment by Myoldmopar** [comment]
@xuanluo113 

There are many LaTeX errors that need to be fixed: https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-dd66a7411d98e462870b28a940b8722575ef49d1-x86_64-Linux-Ubuntu-18.04-gcc-7.5.html

You are using an unknown macro `\degree`.  I can't recall where we landed on degrees in the LaTeX, and if we have a special macro already available, but it obviously doesn't like this one.  I'd just look in the other LaTeX files and see if you can find what we do elsewhere.

**Comment by mjwitte** [comment]
Some info about units may be found [here](https://github.com/NREL/EnergyPlus/wiki/Documentation) in the wiki. Not crystal clear, but it helps.

**Comment by xuanluo113** [comment]
@Myoldmopar @Myoldmopar 
Thanks. A few of the other files use °C/°F as plain text. Others use `\si\celsius` based on the siunitx package, but that's SI only. So I use `\si\celsius` and °F here. Hope that works for CI.

**Comment by Myoldmopar** [comment]
Confirmed the fixed documentation, your section looks good, including the degree symbols.  Thanks for this fix!  Merging.

**Pull request merged**